### ISSUE TITLE
name disambiguation in recent project

### ISSUE THREAD

**Issue by 3nids** [opengisch/QField#602]
if 2 recent projects with the same were loaded recently there is no way to distinguish them

**Comment by signedav** [comment]
![WhatsApp Image 2019-11-05 at 13 32 28 (1)](https://user-images.githubusercontent.com/28384354/68302191-ce5a9900-00a1-11ea-9d99-14c6193e2fc6.jpeg)
Do you mean like this @3nids ?

What would be a sollution? Add the path (or part of it) in grey font next to it?

**Comment by 3nids** [comment]
yep, basically what do Qt Creator, showing the minimum part of the paths to have unambiguous names.

**Comment by signedav** [comment]
Okay, thanks.
I would like that as well.
Still I think it's an enhancement and not a bug.
